src/AppInstallerCLICore/Workflows/PortableFlow.cpp

```diff
@@ -362,8 +572,26 @@ namespace AppInstaller::CLI::Workflow
 
         // Reset termination to allow for ReportInstallResult to process return code.
         context.ResetTermination();
+    }
 
-        // TODO: create subcontext for uninstall
+    void PortableUninstallImpl(Execution::Context& context)
+    {
+        try
+        {
+            context.Reporter.Info() << Resource::String::UninstallFlowStartingPackageUninstall << std::endl;
+
+            context <<
+                RemovePortableExe <<
+                RemoveInstallDirectory <<
+                RemovePortableSymlink;
+
+            context.Reporter.Info() << Resource::String::UninstallFlowUninstallSuccess << std::endl;
```

Comment:
You will print success even if a task has failed.

Reply:
Changed to only printing success of the operationReturnCode is 0